tools/tgs_scripts/InstallDeps.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#find out what we have (+e is important for this)
+set +e
+has_git="$(command -v git)"
+has_cargo="$(command -v ~/.cargo/bin/cargo)"
+has_sudo="$(command -v sudo)"
+has_youtubedl="$(command -v youtube-dl)"
+has_pip3="$(command -v pip3)"
+set -e
+set -x
+
+# install cargo if needed
+if ! [ -x "$has_cargo" ]; then
+	echo "Installing rust..."
+	curl https://sh.rustup.rs -sSf | sh -s -- -y
+	. ~/.profile
+fi
+
+# apt packages, libssl needed by rust-g but not included in TGS barebones install
+if ! ( [ -x "$has_git" ] && [ -f "/usr/lib/i386-linux-gnu/libssl.so" ] ); then
+	echo "Installing apt dependencies..."
+	if ! [ -x "$has_sudo" ]; then
+		dpkg --add-architecture i386
+		apt-get update
+		apt-get install -y lib32z1 git pkg-config libssl-dev:i386 libssl-dev libssl1.1:i386
```

Comment:
```suggestion
		apt-get install -y lib32z1 git pkg-config libssl-dev:i386 libssl-dev gcc-multilib zlib1g-dev:i386
```
i. e. removing libssl 1.1, it's probably already covered by libssl-dev:i386. No idea if 64-bit libssl-dev is needed. gcc-multilib & zlib were both needed to resolve linker errors for me

Reply:
gcc-multilib is already a TGS dependency

Reply:
TGS starts without it for me